portfolio/src/main/webapp/hiddentalent.html

```diff
@@ -19,29 +20,43 @@ <h1>My Hidden Talent</h1>
       </nav>
       <p>One hidden talent that I have not told is that I...</p>
       <button onclick="revealHiddenTalent()">Reveal</button>
-      <div id="hiddentalent-container"></div>
+      <div id="hiddentalent-container">
+        <p>Can create music mashups!<br><br>The inspiration came to me from listening to hundreds of music mashups on
+          YouTube. People were mashing vaporwave and hip hop together like it was peanut butter and  
+          jelly. They were mashing video game soundtracks and pop songs together like mash 
+          potatoes and gravy. I was entranced by the musical artistry.<br><br>I have a playlist of over 50 of my favorite music mashups. 
+          I created my first music mashup in late September 2018. I spent weeks using 
+          the program Audacity making sure it sounded as fluid as possible, with 
+          little training going into it. It may not have sounded the best, but I was thrilled 
+          to have created a mashup that I wanted, and had sounded coherent. The audio 
+          file below contains one of my mashups, 
+          called "Highest in the Woods Instrumental". Feel free to listen if you would like!
+      </div>
       <img src="/images/rollsafe.gif" alt="Roll Safe Dude Gif." class="center" id="rollsafe">
       <audio id="music" controls>
         <source src="/audio/Highest in the Woods Instrumental.wav">
         Your browser does not support the audio tag.
       </audio>
-      <form action="/data" method="POST">
-        <p><b>Want to Leave a Comment?</b>
-        <br/>
-        <label for="name">Name:</label>
-        <input type="text" name="name">
-        <br/>
-        <label for="email">Email:</label>
-        <input type="email" name="email">
-        <br/>
-        <label for="comment">Comment:</label>
-        <textarea name="comment"></textarea>
-        <br/><br/>
-        <input type="submit" />
-      </form>
+      <hr id="divider">
+      <p id="unauth">Click <a href="/login"> here</a> to login and see/post comments.
```

Comment:
I noticed in our pair programming session that when this element was hidden it still took up space. There is some CSS setting that will also make it free up the space it is holding (same from the "requiresauth" elements below). Can you find whatever that setting is and in addition to toggling visibility also toggle the space reservation?